app/views/welcome/index.html.erb

```diff
@@ -1,10 +1,3 @@
-<h1>This is the MSP-Pathfinders Registration system</h1>
-<p id="notice"><%= notice %></p>
-<%= link_to 'Event Listings', events_path %>
-<br>
-<%= link_to 'Scenario Listings', scenarios_path %>
-<br>
-<div class="central" style="width: 100%;">
-  <!-- TODO - add Register (Paypal)! -->
-  <!-- TODO - add Sign Up (Registration Site)! -->
-</div>
\ No newline at end of file
+<h1>The MSP Pathfinders Registration System</h1>
```

Comment:
Actually, shouldn't that be 10,000 Lakes Gaming Registration System?

Not 100% sure, but one entity (the company) is a legal entity... the other is not.